%YAML:1.0

depth_topic: /camera/depth/image_raw

#camera params
fx: 544.473290
fy: 544.473290
cx: 320.000000
cy: 240.000000
height: 480
width: 640

use_extrinsic: 0